docs/classes/IExecVoucherModule.md

```diff
@@ -0,0 +1,183 @@
+[iexec](../README.md) / [Exports](../modules.md) / IExecVoucherModule
+
+# Class: IExecVoucherModule
+
+module exposing voucher methods
+
+## Hierarchy
+
+- [`IExecModule`](IExecModule.md)
+
+  ↳ **`IExecVoucherModule`**
+
+## Table of contents
+
+### Constructors
+
+- [constructor](IExecVoucherModule.md#constructor)
+
+### Properties
+
+- [config](IExecVoucherModule.md#config)
+
+### Methods
+
+- [authorizeRequester](IExecVoucherModule.md#authorizerequester)
+- [getVoucherAddress](IExecVoucherModule.md#getvoucheraddress)
+- [revokeRequesterAuthorization](IExecVoucherModule.md#revokerequesterauthorization)
+- [showUserVoucher](IExecVoucherModule.md#showuservoucher)
+- [fromConfig](IExecVoucherModule.md#fromconfig)
+
+## Constructors
+
+### constructor
+
+• **new IExecVoucherModule**(`configOrArgs`, `options?`): [`IExecVoucherModule`](IExecVoucherModule.md)
+
+Create an IExecModule instance using an IExecConfig like
```

Comment:
It seems incomplete or lacks clarification

Reply:
removed unclear formulation 93b4af336ae81d99cf7ca2df436237ff164da83b